packages/material-ui/src/InputBase/InputBase.js

```diff
@@ -139,6 +139,8 @@ export const styles = theme => {
   };
 };
 
+const ariaRegExp = /^aria-/;
```

Comment:
this is not related to this PR but the same treatment for `data-` attributes would be also good